src/test/java/org/broadinstitute/hellbender/engine/ReadWalkerGCSSupportIntegrationTest.java

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.PrintReads;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Tests to prove that we can access and query inputs on Google Cloud Storage (GCS) in ReadWalkers.
+ */
+public class ReadWalkerGCSSupportIntegrationTest extends CommandLineProgramTest {
+
+    private static final String TEST_BAM_ON_GCS = "org/broadinstitute/hellbender/engine/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.10000000-10000020.with.unmapped.bam";
```

Comment:
Fair enough! We should be able to exercise the prefetcher without taking too long, though, by reading only a small part of the large file and keeping the prefetch size small.